src/test/parse/tokenize_definition.spec.ts

```diff
@@ -67,6 +67,13 @@ describe("tokenizeDefinition()", () => {
         expect(result).to.have.ordered.members(["arg:string", "arg2:Callable[[], str]", "-> str"]);
     });
 
+    it("should tokenize pep604 parameter and return types", () => {
```

Comment:
Can you add a test to check that it can tokenize the return type as well?

Reply:
Good catch that would have actually broken every thing 😅